tests/meson.build

```diff
@@ -153,6 +153,18 @@ if gtest_dep.found()
     test('unittest_filter_armnn', unittest_filter_armnn, env: testenv)
   endif
 
+  # TVM unittest
+  if tvm_support_is_available
+    unittest_filter_tvm = executable('unittest_filter_tvm',
+      join_paths('nnstreamer_filter_tvm', 'unittest_filter_tvm.cc'),
+      dependencies: [glib_dep, gst_dep, nnstreamer_dep, gtest_dep, dlpack_dep, tvm_dep],
```

Comment:
Do you really need both dlpack and tvm for the unittest?
If unittest-tvm --depends-on--> tvm-subplugin only, you should not depend on dlpack/tvm directly.

Reply:
That's right. Fixed!